cpp/common/src/codingstandards/cpp/Bitwise.qll

```diff
@@ -0,0 +1,18 @@
+/**
+ * A library for addressing issues in bitwise operator modelling in our database schema.
+ */
+
+import cpp
+
+/**
+ * A binary bitwise assign operation, excluding += and -= on pointers, which seem to be erroneously
+ * included.
+ */
+class AssignBitwiseOperationFixed extends AssignBitwiseOperation {
+  AssignBitwiseOperationFixed() {
+    // exclude += and -= on pointers, which seem to be erroneously included
+    // in the database schema
+    not this instanceof AssignPointerAddExpr and
+    not this instanceof AssignPointerSubExpr
+  }
+}
```

Comment:
```suggestion
/**
 * A library for addressing issues in bitwise operator modelling in our database schema.
 */

import cpp as cpp

module Bitwise {
  /**
   * A binary bitwise assign operation, excluding += and -= on pointers, which seem to be erroneously
   * included.
   */
  class AssignBitwiseOperation extends cpp::AssignBitwiseOperation {
    AssignBitwiseOperation() {
      // exclude += and -= on pointers, which seem to be erroneously included
      // in the database schema
      not this instanceof cpp::AssignPointerAddExpr and
      not this instanceof cpp::AssignPointerSubExpr
    }
  }
}
```

The suffix `Fixed` is ambiguous so I would like to suggest to have our own complete version and document the reasons for having our own version.

Reply:
Agreed and changed.